### PR TITLE
added authorship and contributor roles pages (issue 17), solved issue…

### PR DESCRIPTION
**Pull request by paolap**
added authorship page in concepts, to clarify who is an author who is a contributor and introduce authorship policy
Add contributor roles page in tech section, this is linked from the authorship page, both are linked to issue #17.

solved issue #46 and #47 

finally fixed a couple of broken references

### CODE REVIEW

File: Governance/manage/manage-project.md
Comment (hot007) [inline Governance/manage/manage-project.md:60]:
```suggestion
As a result, data can be transferred to count against the quota of a different project than where it lives, and cause the target project to reach its storage quota. This can cause unforeseen issues for data projects who typically have a lot of users and can result in automatic dataset update to fail for lack of storage.
```

File: Governance/tech/title.md
Comment (hot007) [inline Governance/tech/title.md:10]: who is "them"?
I guess also it helps projects track their research outputs beyond publications.

**Review by hot007 — APPROVED** [review]
One typo, one comment. Approving assuming you'll fix the typo :)

File: Governance/tech/title.md
Comment (paolap) [reply Governance/tech/title.md:10]: I'll remove "for them" in my mind "them" were the potential users :-)

File: Governance/concepts/authorship.md
Comment (chloemackallah) [inline Governance/concepts/authorship.md:9]: I think we could rethink the use of {note}s, esp on this page where they kind of seem out of place in the formatted version. This one is more of a footnote, but appears as a highlighted section of importance.

File: Governance/concepts/authorship.md
Comment (chloemackallah) [inline Governance/concepts/authorship.md:23]: With this {tip} and the following {note}, the URL hyperlinks do not work under these blocked out sections, but appear as typed. 
They also format as a single line that you need to scroll horizontally on, which makes them quite unwieldy.

**Review by chloemackallah — CHANGES_REQUESTED** [review]
All good re content; its just the note blocks that don't format very well.

File: Governance/concepts/authorship.md
Comment (paolap) [reply Governance/concepts/authorship.md:9]: thanks I'll double checked, that nearly looks like something got lost!!

File: Governance/concepts/authorship.md
Comment (paolap) [reply Governance/concepts/authorship.md:23]: ok, I'll double checked this too, I thought it worked in mine so it might be a case of a missing library, otherwise I'll change them

File: Governance/concepts/authorship.md
Comment (paolap) [reply Governance/concepts/authorship.md:9]: My bad I started writing the note and never finished, it's completed now. I kind use notes also to break down too much text. I removed the second Notes that could have just been added as a sentence.

File: Governance/concepts/authorship.md
Comment (paolap) [reply Governance/concepts/authorship.md:23]: They work for me, so it might be a matter of missing extensions in the conda env. When you say they don't work I assume that you tried to build it locally. I would leave it until we merge and if also the main branch has the same issue, I will check which specific extension is needed.

Comment (paolap) [reply Governance/concepts/authorship.md:23]: @chloemackallah I pushed the changes if you want to check again and let me know if the formatting is better

Comment (chloemackallah) [reply Governance/concepts/authorship.md:23]: Looks better. The URLs still aren't linking properly, but perhaps they will once merged and formatted properly in the book rather than just 'viewed' in the PR.

**Review by chloemackallah — APPROVED** [review]

File: Governance/concepts/authorship.md
Comment (paolap) [reply Governance/concepts/authorship.md:23]: URLs work in the book :-)

**Pull request merged**
